event_model/documents/stream_datum.py

```diff
@@ -18,13 +18,6 @@ class StreamDatum(TypedDict):
             "be contiguous for a given stream.",
         ),
     ]
-    datum_kwargs: Annotated[
-        Dict[str, Any],
-        Field(
-            description="Arguments to pass to the Handler to retrieve one "
-            "quanta of data",
-        ),
-    ]
     event_count: Annotated[
```

Comment:
`event_count` and `event_offset` can be deleted, they are now represented by `seq_nums`